pkg/cmd/server/api/v1/conversions.go

```diff
@@ -17,6 +17,15 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 			if len(obj.APILevels) == 0 {
 				obj.APILevels = internal.DefaultOpenShiftAPILevels
 			}
+			if len(obj.JenkinsPipelineConfig.Namespace) == 0 {
+				obj.JenkinsPipelineConfig.Namespace = "openshift"
+			}
+			if len(obj.JenkinsPipelineConfig.TemplateName) == 0 {
+				obj.JenkinsPipelineConfig.TemplateName = "jenkins"
+			}
+			if len(obj.JenkinsPipelineConfig.ServiceName) == 0 {
+				obj.JenkinsPipelineConfig.ServiceName = "jenkins-ci"
```

Comment:
That's why this name should probably be "ci".  We're installing the "ci"
component to your project (jenkins is a detail, a good one, but still a
detail)

On Wed, Mar 30, 2016 at 4:53 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/cmd/server/api/v1/conversions.go
> <https://github.com/jimmidyson/origin/pull/2#discussion_r57961741>:
>
> > @@ -17,6 +17,15 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
> >  			if len(obj.APILevels) == 0 {
> >  				obj.APILevels = internal.DefaultOpenShiftAPILevels
> >  			}
> > +			if len(obj.JenkinsPipelineConfig.Namespace) == 0 {
> > +				obj.JenkinsPipelineConfig.Namespace = "openshift"
> > +			}
> > +			if len(obj.JenkinsPipelineConfig.TemplateName) == 0 {
> > +				obj.JenkinsPipelineConfig.TemplateName = "jenkins"
> > +			}
> > +			if len(obj.JenkinsPipelineConfig.ServiceName) == 0 {
> > +				obj.JenkinsPipelineConfig.ServiceName = "jenkins-ci"
>
> what happens when the underlying implementation engine for the other
> pipeline strategy is not jenkins?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jimmidyson/origin/pull/2/files/3ed08d495dd391ab47317d2946c2695da23b317b#r57961741>
>